marklogic-data-hub-central/ui/src/components/job-response/job-response.tsx

```diff
@@ -53,15 +46,10 @@ const JobResponse: React.FC<Props> = (props) => {
          }) : []; */
         //const successfulStep = successfulSteps[successfulSteps.length - 1];
         //setLastSuccessfulStep(successfulStep);
-        if (durationInterval) {
-          clearInterval(durationInterval);
-          setDurationInterval(null);
-        }
         if (isRunning(response.data)) {
-          setDurationInterval(setInterval(() => {
-            const duration = durationFromDateTime(response.data.timeStarted);
-            setJobResponse(Object.assign({}, response.data, {duration}));
-          }, 1000));
+          const duration = durationFromDateTime(response.data.timeStarted);
+          setJobResponse(Object.assign({}, response.data, {duration}));
```

Comment:
I checked and for a small ingestion of data in a full flow we are calling the backend less than once per second.
The story still mentions at least 3 second. Was there a performance check to evaluate the right wait time?